dev-libs/libbase58/libbase58-0.1.4.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A library for encoding/decoding base58"
+DESCRIPTION="libbase58 provides encoding and decoding for the base58 format, \
+used in Bitcoin addresses."
+HOMEPAGE="https://github.com/bitcoin/libbase58"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libbase58/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="97e8c3387fd4ed90208a8101e658743efbb6e21714e594f0828e6414f92c0b1d"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
+    PATCHES="${portVersionedName}.patchset"
+fi
+
+PROVIDES="
+    libbase58${secondaryArchSuffix} = $portVersion
+    lib:libbase58${secondaryArchSuffix} = $portVersion
+    devel:libbase58${secondaryArchSuffix} = $portVersion
+"
+
+REQUIRES="
+    haiku${secondaryArchSuffix}
```

Comment:
You can (and probably should) replace `${secondaryArchSuffix}` by `$secondaryArchSuffix` everywhere *except*, of course, in `haiku${secondaryArchSuffix}_devel`.
But that's just a recommendation. We still have some unneeded curly braces here and there on some recipes.